src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -346,6 +346,11 @@ ManageCLIError.InvalidGlacierOperation = Object.freeze({
     message: 'only "migrate", "restore" and "expiry" subcommands are supported',
     http_code: 400,
 });
+ManageCLIError.InvalidSupplementalGroupsList = Object.freeze({
+    code: 'InvalidSupplementalGroupsList',
+    message: 'supplemental groups must be a list of group ids (positive integers)',
```

Comment:
I think we need to address the case of 0 in this message.
0 is neither positive and negative, but can be one of the groups, right?
Something like:
```suggestion
    message: 'supplemental groups must be a list of group ids (group id is zero or a positive integer)',
```